aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -83,10 +100,36 @@ module aptos_token::property_map {
     ): (String, PropertyValue) {
         let found = contains_key(list, key);
         assert!(found, EPROPERTY_NOT_EXIST);
-
+        let (found, ind) = vector::index_of(&mut list.keys, key);
+        assert!(found, EPROPERTY_NOT_EXIST);
```

Comment:
I think this could be addressed by adding just a check on contains.

Reply:
1. property_map is built on `simplemap`, which is not copyable. I also don't want to make SimpleMap copyable since it is also used for storing signercap.
2. I mainly need the `index`. found is a byproduct and used for the assert